### PR TITLE
Simplify use

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
 
         # This is needed to link the correct project.
         # Get it via: symfony project:info command
-        SYMFONY_PROJECT_ID: ${{ secrets.SYMFONY_PROJECT_ID }}
+        SYMFONY_PROJECT: xxxxxxxxxxxxx
```
Comment (tucksaun) [inline README.md:32]: this is not considered a sensible value

**Review by OskarStark — APPROVED** [review]
cc @localheinz

File: README.md (hunk 1 above)
Comment (localheinz) [reply README.md:32]: @tucksaun 

Why remove the `_ID` suffix?

Running

```
$ symfony link --help
```

yields

``` 
Usage:
  symfony project:link [options] [--] [<id>]

Arguments:
  id

Options:
  --force, -f  Override any previous link [default: false]

```

so I believe the `_ID` suffix makes sense here.

What do you think?

**Comment by tucksaun** [comment]
`SYMFONY_PROJECT` is automatically read by the tool and make useless the
`link` step
-- 
---
Tugdual Saunier

**Comment by localheinz** [comment]
@tucksaun 

Is that documented somewhere?

**Review by localheinz — APPROVED** [review]

**Comment by tucksaun** [comment]
No, will do
-- 
---
Tugdual Saunier

**Comment by localheinz** [comment]
Thank you, @tucksaun!

**Pull request merged**